languages/javascript/reference/keywords/export.md

```diff
@@ -0,0 +1,29 @@
+# `export`
+
+The `export` keyword is used to define elements of the current file, that can be imported into other files using the [`import`][concept-import] keyword.
+
+There are two types of exports, the default export and named exports.
+
+## Default export
+
+The default export is marked using the `default` modifier:
+
+```js
+export default class XYZ {}
+```
+
```

Comment:
Can we add a line or two that links to tooling such as `tsc` and `babel` allowing for synthetic default export (`exports = x` becomes `export default`, as well options for translating `export default` to a named export with the name `default`)., as well as a section about difference between the `export` keyword and `commonjs` exports?

Reply:
sure

Reply:
This is not C# or Ruby, yet I often link to those when explaining something, such as interop, similarities or differences. In this particular case I think it makes sense to link to `tsc` because it _produces_ JavaScript. However, I am open to have my opinion swayed.